src/components/RewardsWrapper/index.tsx

```diff
@@ -38,6 +38,14 @@ export function RewardsWrapper({
   const questMeta = questQuery?.data
   const rewardsData = rewardsQuery?.data
 
+  if (questQuery.isError) {
+    logError(`Error in quest query ${questQuery.error}`)
+  }
+
+  if (rewardsQuery.isError) {
+    logError(`Error in rewards query ${rewardsQuery.error}`)
+  }
+
```

Comment:
This will log on every render, I believe you did this because the useQuery hook doesn't have an onError callback. The recommended way of doing this is this way https://tkdodo.eu/blog/breaking-react-querys-api-on-purpose#a-bad-api

```ts
const queryClient = new QueryClient({
  queryCache: new QueryCache({
    onError: (error) = console.error(error)
  }),
})
```

In our case, this won't work because we don't have `logError` at that point in the code. What do you think about using a `useEffect` instead?

Reply:
I'm fine with that. As long as we can get the error message. On the store, it was silently failing because of an outdated chains package that didn't have apechain so as long as we can log it, it will make debugging easier in the future.